src/python/pants/rules/core/test.py

```diff
@@ -79,9 +79,19 @@ class TestOptions(GoalSubsystem):
   @classmethod
   def register_options(cls, register) -> None:
     super().register_options(register)
-    register('--debug', type=bool, default=False,
-             help='Run a single test target in an interactive process. This is '
-                  'necessary, for example, when you add breakpoints in your code.')
+    register(
+      '--debug',
```

Comment:
Thoughts on if it makes sense to run coverage with `--debug` at the same time? CC @codealchemy. This would complicate things that the `debug_python_test` would somehow need to capture the resulting `.coverage`. For now, I'd punt on it and possibly do validation in `rules/core/test.py` to ensure that both aren't specified at the same time.

Generally, Alex what do you think about this use case?

Reply:
IMO: wait-for-or-implement #8923 to avoid a combinatorial explosion here. I'm hoping to look at it again this weekend.

Reply:
I have no idea what the coverage instrumentation might do with pdb, if anything. I don't think it makes sense to run both at once, though maybe someone can think up a use case for that?